sklearn/model_selection/tests/test_search.py

```diff
@@ -1220,7 +1220,7 @@ def test_search_cv_results_none_param():
     X, y = [[1], [2], [3], [4], [5]], [0, 0, 0, 0, 1]
     estimators = (DecisionTreeRegressor(), DecisionTreeClassifier())
     est_parameters = {"random_state": [0, None]}
-    cv = KFold(random_state=0)
+    cv = KFold()
```

Comment:
Why is this okay in tests now? Each instance will have a different seed, therefore the tests are not going to be deterministic w/o the seed. I'm confused.

Reply:
shuffle is False by default ;)

That's why I want to raise an error when shuffle is False but random_state isn't the default, cf https://github.com/scikit-learn/scikit-learn/pull/15177/files#r338127979